### DISCUSSION TITLE
Grabaciones de las clases

### DISCUSSION THREAD

**Discussion by Quercus28** [rramosp/ai4eng.v1#73] [00 - General]
Buen día, profesor.
¿Dónde puedo encontrar las grabaciones de las clases pasadas? 
Saludos.

**Comment by rramosp** [comment]
hola quercus, en la página del curso "2021.2 udea" está toda la información

El dom, 28 nov 2021 a las 9:56, Quercus Humboldtii (<
***@***.***>) escribió:

> Buen día, profesor.
> ¿Dónde puedo encontrar las grabaciones de las clases pasadas?
> Saludos.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/73>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKWVH3EQG65OSFHKCLUOI7LPANCNFSM5I5MG7DA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by andresfzt** [comment]
Buen día profesor Ramos.

En el video '2.15 - Pandas - Reparación de datasets' (https://www.youtube.com/watch?v=MceJd7-czHM&t=620s) en el segundo 2:35 cuando se menciona lo de obtener el mes de la serie de datos se nota que el resultado es extraño porque el resultado es 2 (Febrero) a pesar de que el dato que se estaba revisando era el segundo del dataframe. 
Creo que esto influye en el resultado al final del video, donde se grafica la columna Brebeouf. Al definir el formato de fecha adecuado en el comando to_datetime la gráfica es totalmente diferente, los datos NaN al inicio del dataframe no se grafican y después si existen datos.

Muchas gracias por el curso.

**Comment by rramosp** [comment]
gracias andrés por tu contribución, voy a revisarlo para corregirlo si
procede.

El lun, 29 nov 2021 a las 22:04, andresfzt ***@***.***>)
escribió:

> Buen día profesor Ramos.
>
> En el video '2.15 - Pandas - Reparación de datasets' (
> https://www.youtube.com/watch?v=MceJd7-czHM&t=620s) en el segundo 2:35
> cuando se menciona lo de obtener el mes de la serie de datos se nota que el
> resultado es extraño porque el resultado es 2 (Febrero) a pesar de que el
> dato que se estaba revisando era el segundo del dataframe.
> Creo que esto influye en el resultado al final del video, donde se grafica
> la columna Brebeouf. Al definir el formato de fecha adecuado en el comando
> to_datetime la gráfica es totalmente diferente, los datos NaN al inicio del
> dataframe no se grafican y después si existen datos.
>
> Muchas gracias por el curso.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/73#discussioncomment-1719683>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIPE2ZKEUHC2BAMFYFLUOQ5KZANCNFSM5I5MG7DA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
